common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -155,14 +171,15 @@ div.problem {
   }
 
   .question-description {
-    @include margin(($baseline*0.75), 0);
+    font-size: 0.9em;
```

Comment:
I see all these em values throughout the file and I know you mentioned they correspond to the correct font sizes.  I think @cahrens also had questions about these values.

1. Where are they defined?
2. If they should be used all over should be make them variables in a common file and use the variables throughout?